storage/containerd.md

```diff
@@ -0,0 +1,58 @@
+---
+title: containerd image store with Docker Engine
+keywords: containerd, snapshotters, image store, docker engine
+description: Enabling the containerd image store on Docker Engine
+---
+
+> **Note**
+>
+> The containerd image store is an experimental feature of Docker Engine.
+> If you're using Docker Desktop, don't use the instructions on this page. See
+> [containerd image store with Docker Desktop](../desktop/containerd/index.md).
+
+containerd, the industry-standard container runtime, uses snapshotters instead
+of the classic storage drivers for storing image and container data.
+While the `overlay2` driver still remains the default driver for Docker Engine,
+you can opt in to using containerd snapshotters as an experimental feature.
+
+To learn more about the containerd image store and it's benefits, refer to
+[containerd image store on Docker Desktop](../desktop/containerd/index.md).
+
+## Enable containerd image store on Docker Engine
+
+Switching to containerd snapshotters causes you to temporarily lose images and
+containers created using the classic storage drivers.
+Those resources still exist on your filesystem, and you can retrieve them by
+disabling the containerd snapshotters feature.
+
+The following steps show how to enable the containerd snapshotters feature.
```

Comment:
On first reading, `show how` makes me think I'll be seeing a demonstration or something. Suggestion to change:
```suggestion
The following steps explain how to enable the containerd snapshotters feature.
```